### PR TITLE
:imp: copy live styles

### PR DESCRIPTION
**Pull request by ajthomascouk**

### CODE REVIEW

**Comment by ajthomascouk** [comment]
Good point about the `#fff`. Changed to `$white`

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 0.23.1 / TBD
 ===================
+- Install `greenkeeper-lockfile` with `npm` rather than `yarn` as latest version of `yarn` isn't working see https://github.com/greenkeeperio/greenkeeper-lockfile/issues/98
```
Comment (st3v3nhunt) [inline CHANGELOG.md]: This entry shouldn't be here any more, can you remove please?

Comment (ajthomascouk) [reply CHANGELOG.md]: Well spotted. Updated.

Comment (st3v3nhunt) [reply CHANGELOG.md]: This line is still in the file, can it be deleted please?

Comment (ajthomascouk) [reply CHANGELOG.md]: Sorry my comment was for the ie styles.

**Pull request merged**